libcerf/plan.sh

```diff
@@ -0,0 +1,29 @@
+pkg_name=libcerf
+pkg_origin=core
+pkg_version="1.5"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source="http://apps.jcns.fz-juelich.de/src/$pkg_name/$pkg_name-$pkg_version.tgz"
+pkg_shasum="e36dc147e7fff81143074a21550c259b5aac1b99fc314fc0ae33294231ca5c86"
+# pkg_deps=(core/glibc)
```

Comment:
This should be uncommented. It ends up linked correctly if you don't included it, but it glibc does show up under `ldd` for libcerf.so so it's better to have it as an explicit dependency.